dashboard/app/views/plc/tasks/view/_script_completion_task.html.haml

```diff
@@ -0,0 +1,3 @@
+%p This is a Script Completion Task
+
+%p The user is expected to complete script id #{script_id}
```

Comment:
Reworked the page - it'll handle a Script Completion Level that doesn't have a script id